### PR TITLE
feat: `spark routes` option to sort by handler

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
Supersedes #6415

- add option `-h` to sort by handler

```console
$ php spark routes

CodeIgniter v4.2.11 Command Line Tool - Server Time: 2022-12-24 02:10:28 UTC+00:00

+--------+------------------------------+------+---------------------------------------------+----------------+---------------+
| Method | Route                        | Name | Handler                                     | Before Filters | After Filters |
+--------+------------------------------+------+---------------------------------------------+----------------+---------------+
| GET    | /                            | »    | \App\Controllers\Home::index                |                | toolbar       |
| GET    | bbs/index                    | »    | \App\Controllers\Bbs::index                 |                | toolbar       |
| GET    | bbs/post                     | »    | \App\Controllers\Bbs::post                  |                | toolbar       |
| GET    | form                         | »    | \App\Controllers\Form::index                |                | toolbar       |
| GET    | shop/cart                    | »    | \App\Controllers\Shop\Cart::index           |                | toolbar       |
| GET    | shop/index/([0-9]+)          | »    | \App\Controllers\Shop\Index::index/$1       |                | toolbar       |
| GET    | shop/index/([0-9]+)/([0-9]+) | »    | \App\Controllers\Shop\Index::index/$1/$2    |                | toolbar       |
| GET    | shop/product/([0-9]+)        | »    | \App\Controllers\Shop\Product::index/$1     |                | toolbar       |
| GET    | shop/search                  | »    | \App\Controllers\Shop\Search::index         |                | toolbar       |
| GET    | shop/search/([0-9]+)         | »    | \App\Controllers\Shop\Search::index/$1      |                | toolbar       |
| POST   | bbs/confirm                  | »    | \App\Controllers\Bbs::confirm               |                | toolbar       |
| POST   | bbs/delete/([0-9]+)          | »    | \App\Controllers\Bbs::delete/$1             |                | toolbar       |
| POST   | bbs/insert                   | »    | \App\Controllers\Bbs::insert                |                | toolbar       |
| POST   | form                         | »    | \App\Controllers\Form::index                |                | toolbar       |
| POST   | form/confirm                 | »    | \App\Controllers\Form::confirm              |                | toolbar       |
| POST   | form/send                    | »    | \App\Controllers\Form::send                 |                | toolbar       |
| POST   | shop/add/([0-9]+)            | »    | \App\Controllers\Shop\Cart::add/$1          |                | toolbar       |
| POST   | shop/customer_info           | »    | \App\Controllers\Shop\CustomerInfo::index   |                | toolbar       |
| POST   | shop/confirm                 | »    | \App\Controllers\Shop\CustomerInfo::confirm |                | toolbar       |
| POST   | shop/order                   | »    | \App\Controllers\Shop\Order::index          |                | toolbar       |
+--------+------------------------------+------+---------------------------------------------+----------------+---------------+
```

```console
$ php spark routes -h

CodeIgniter v4.2.11 Command Line Tool - Server Time: 2022-12-24 02:10:32 UTC+00:00

+--------+------------------------------+------+---------------------------------------------+----------------+---------------+
| Method | Route                        | Name | Handler ↓                                   | Before Filters | After Filters |
+--------+------------------------------+------+---------------------------------------------+----------------+---------------+
| POST   | bbs/confirm                  | »    | \App\Controllers\Bbs::confirm               |                | toolbar       |
| POST   | bbs/delete/([0-9]+)          | »    | \App\Controllers\Bbs::delete/$1             |                | toolbar       |
| GET    | bbs/index                    | »    | \App\Controllers\Bbs::index                 |                | toolbar       |
| POST   | bbs/insert                   | »    | \App\Controllers\Bbs::insert                |                | toolbar       |
| GET    | bbs/post                     | »    | \App\Controllers\Bbs::post                  |                | toolbar       |
| POST   | form/confirm                 | »    | \App\Controllers\Form::confirm              |                | toolbar       |
| GET    | form                         | »    | \App\Controllers\Form::index                |                | toolbar       |
| POST   | form                         | »    | \App\Controllers\Form::index                |                | toolbar       |
| POST   | form/send                    | »    | \App\Controllers\Form::send                 |                | toolbar       |
| GET    | /                            | »    | \App\Controllers\Home::index                |                | toolbar       |
| POST   | shop/add/([0-9]+)            | »    | \App\Controllers\Shop\Cart::add/$1          |                | toolbar       |
| GET    | shop/cart                    | »    | \App\Controllers\Shop\Cart::index           |                | toolbar       |
| POST   | shop/confirm                 | »    | \App\Controllers\Shop\CustomerInfo::confirm |                | toolbar       |
| POST   | shop/customer_info           | »    | \App\Controllers\Shop\CustomerInfo::index   |                | toolbar       |
| GET    | shop/index/([0-9]+)          | »    | \App\Controllers\Shop\Index::index/$1       |                | toolbar       |
| GET    | shop/index/([0-9]+)/([0-9]+) | »    | \App\Controllers\Shop\Index::index/$1/$2    |                | toolbar       |
| POST   | shop/order                   | »    | \App\Controllers\Shop\Order::index          |                | toolbar       |
| GET    | shop/product/([0-9]+)        | »    | \App\Controllers\Shop\Product::index/$1     |                | toolbar       |
| GET    | shop/search                  | »    | \App\Controllers\Shop\Search::index         |                | toolbar       |
| GET    | shop/search/([0-9]+)         | »    | \App\Controllers\Shop\Search::index/$1      |                | toolbar       |
+--------+------------------------------+------+---------------------------------------------+----------------+---------------+
```

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: system/Commands/Utilities/Routes.php [hunk 1]
```diff
@@ -68,13 +68,17 @@ class Routes extends BaseCommand
      *
      * @var array
```
Comment (datamweb) [inline system/Commands/Utilities/Routes.php:69]:
```suggestion
     * @var array<string, string>
```

**Review by datamweb — CHANGES_REQUESTED** [review]
Tested, good job!

Screenshot 2022-12-24 135444

**Comment by ddevsr** [comment]
Why not `-s` or `--sort`? In `-s` can be sort by handler or route or anything can be sort?

Like `-s handler` or `-s route`

**Comment by kenjis** [comment]
`-h` for Handler.
The other options seem to be mostly unnecessary, so I went with `h`.

**Review by michalsn — APPROVED** [review]
`-h` make sense since I don't think there will be other functionality than sorting here.
But let's see if there will be more voices for the alternative solution.

**Comment by kenjis** [comment]
@codeigniter4/core-team If everyone is okay, I would like to merge this.

**Review by samsonasik — APPROVED** [review]

**Review by MGatner — APPROVED** [review]
The sort arrow is a nice touch! 🤗

**Pull request merged**